CONTRIBUTING.md

```diff
@@ -250,19 +250,19 @@ And in case we didn't emphasize it enough: we love tests!
 
 NOTE: Partially copied from https://raw.github.com/thoughtbot/factory_girl_rails/master/CONTRIBUTING.md
 
-# Travis CI Tests
+# CI (Github Actions) Tests
 
-We use [Travis CI](https://travis-ci.org/emberjs/ember.js/pull_requests) to test each PR before it is merged.
+We use [GitHub Actions](https://github.com/ember.js/ember.js/actions?query=workflow%3ABuild+event%3Apull_request) to test each PR before it is merged.
```

Comment:
I noticed that the current URL is missing the period and redirects to 404.
```suggestion
We use [GitHub Actions](https://github.com/emberjs/ember.js/actions?query=workflow%3ABuild+event%3Apull_request) to test each PR before it is merged.
```

Reply:
Good catch, thanks!